src/components/search/detailed/AnalysesSearchResults.js

```diff
@@ -0,0 +1,166 @@
+/**
+ *
+ * Display tablular view of detailed analyses search results
+ *
+ * @author sriram
+ *
+ */
+
+import React, { useEffect, useState } from "react";
+
+import { useTranslation } from "i18n";
+
+import { useAnalysesSearchInfinite } from "../searchQueries";
+import SearchError from "./SearchError";
+import SearchResultsTable from "./SearchResultsTable";
+import { getAnalysesSearchQueryFilter } from "../analysesSearchQueryBuilder";
+import searchConstants from "../constants";
+import constants from "../../../constants";
+
+import { formatDate } from "@cyverse-de/ui-lib";
+
+import { ANALYSES_SEARCH_QUERY_KEY } from "serviceFacades/analyses";
+import analysisFields from "components/analyses/analysisFields";
+import { Typography } from "@material-ui/core";
+
+export default function AnalysesSearchResults(props) {
+    const { searchTerm, updateResultCount, baseId } = props;
+    const [analysesSearchKey, setAnalysesSearchKey] = useState(
+        ANALYSES_SEARCH_QUERY_KEY
+    );
+    const [
+        analysesSearchQueryEnabled,
+        setAnalysesSearchQueryEnabled,
+    ] = useState(false);
+
+    const { t } = useTranslation("search");
+    const { t: at } = useTranslation("analyses");
+
+    let analysisRecordFields = analysisFields(at);
+
+    const [order, setOrder] = useState(constants.SORT_DESCENDING);
+    const [orderBy, setOrderBy] = useState(analysisRecordFields.START_DATE.key);
+
+    const {
+        status,
+        data,
+        isFetchingMore,
+        fetchMore,
+        canFetchMore,
+        error,
+    } = useAnalysesSearchInfinite(
+        analysesSearchKey,
+        analysesSearchQueryEnabled,
+        (lastGroup, allGroups) => {
+            const totalPages = Math.ceil(
+                lastGroup?.total / searchConstants.DETAILED_SEARCH_PAGE_SIZE
+            );
+            if (allGroups.length < totalPages) {
+                return allGroups.length;
+            } else {
+                return false;
+            }
+        }
+    );
```

Comment:
if there are more pages to get, then return a `truthy` value else return a `falsey` value.

Reply:
Oops. I misspoke, if there are more pages, we should return the next page number otherwise return false.

Reply:
And `lastgroup` shouldn't be null / nundefined unless we have miscalculated the `totalPages` 🤔 .